internal/driver/glfw/menu_bar_test.go

```diff
@@ -526,3 +526,18 @@ func TestMenuBar_Toggle(t *testing.T) {
 		test.AssertRendersToMarkup(t, "menu_bar_toggle_deactivated.xml", c)
 	})
 }
+
+type notFocusableButton struct {
+	widget.Label
+	f func()
+}
+
+func newNotFocusableButton(l string, f func()) *notFocusableButton {
+	n := &notFocusableButton{f: f}
+	n.Label.Text = l
+	return n
+}
+
+func (n *notFocusableButton) Tapped(e *fyne.PointEvent) {
+	n.f()
+}
```

Comment:
This shouldn't be needed now 🤔

Reply:
As above, can you help me understand why?